### ISSUE TITLE
Proposal: Add css support with help of kotlin-css-jvm

### ISSUE THREAD

**Issue by vlipovetskii** [mvysny/karibu-dsl#70]
```
inline fun HasStyle.css(block: CssBuilder.() -> Unit) =
    CssBuilder().apply {
        block()
        declarations.forEach { (key, value) ->
            style[key.hyphenize()] = value.toString()
        }
    }

```
Sample of usage:
```
css {
   backgroundColor = Color.transparent
}

```

Please note: It will require to add depenedency on [kotlin-css-jvm](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-css-jvm)

@mvysny If it's ok for you I will prepare PR.

**Comment by mvysny** [comment]
The `css{}` syntax looks good, let's do it 💪  The kotlin-css-jvm is reasonably small: the jar is 332kb long and doesn't pull transitive dependencies: https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-css-jvm/1.0.0-pre.847 .

**Comment by mvysny** [comment]
Closed by #71 , will be released in Karibu-DSL 2.2.1
